solr/core/src/test/org/apache/solr/search/QueryEqualityTest.java

```diff
@@ -1364,12 +1420,12 @@ public void testQueryKNN() throws Exception {
    * for coverage sanity checking
    *
    * @see #testParserCoverage
-   * @see #assertQueryEquals
+   * @see #assertQueryEqualsAndReturn
```

Comment:
If I am reading it right, these are 4 different test cases but currently run as one?
I would prefer to see them separate, because right now if I want to run and debug the last one I'm forced to run all of them?
I may be wrong, so feel free to elaborate!